lib/helpers/curl.js

```diff
@@ -26,18 +26,21 @@ module.exports = {
     var str;
 
     method = method || 'GET';
+
+    if (data && method.toLowerCase() === 'get') {
+      uri += this.buildQueryString(data);
+    }
+
     str = ['curl', this.buildFlag('X', method.toUpperCase(), 0, ''), '"' + uri + '"'].join(' ');
 
-    _.each(headers, function(val, header) {
-      flags.push(this.buildFlag('H', header + config.HEADER_SEPARATOR + val, 5));
-    }, this);
+    if (headers) {
+      _.each(headers.properties, function(val, header) {
```

Comment:
Think about this a little more. If we're going to make header properties follow the json schema property spec, it'd more flexible to use `ExampleDataExtractor.extract()` to generate the key/values for the headers before building the flags